types/react-relay/tsconfig.json

```diff
@@ -19,8 +19,6 @@
         "jsx": "react"
     },
     "files": [
-        "index.d.ts",
-        "test/react-relay-tests.tsx",
-        "test/relay-experimental-tests.tsx"
+        "index.d.ts"
```

Comment:
think we may need to expose the /legacy and /hooks here (if those marry up to the export sub-modules of the packages)

Reply:
Ah, yeah! Thanks for catching that.

I wonder if we should expose all of the files individually as well, since it is possible to import individual files.

Reply:
Although, I wonder why that wasn't required for the previous set of types? The readme makes it seem like, since these files are accessible from index.d.ts, I should be good... fingers crossed.

Reply:
I wonder if something changed. I get 

```
Only index.d.ts may be listed explicitly in tsconfig's "files" entry.
Other d.ts files must either be referenced through index.d.ts, tests, or added to OTHER_FILES.txt.
```